WordPress/Classes/ViewRelated/Site Creation/Site Name/SiteNameStep.swift

```diff
@@ -1,24 +1,35 @@
 import Foundation
+import UIKit
 
 /// Site Creation: Allows creation of the site's name.
 final class SiteNameStep: WizardStep {
     weak var delegate: WizardDelegate?
     private let creator: SiteCreator
 
-    private(set) lazy var content: UIViewController = {
-        return SiteNameViewController(creator: creator)
-    }()
+    var content: UIViewController {
```

Comment:
Could you help me understand why this is a computed property over lazy? Was it to ensure everything was rebuilt in case a difference vertical was selected?

Reply:
> Was it to ensure everything was rebuilt in case a difference vertical was selected?

That was certainly one of the effects. More generally, I don't think there is any reason to hold a view in memory when it's outside its view hierarchy. Keeping that `lazy` property would do that, and keep `SiteNameViewController` in memory even when users navigate back to the site intent.

Reply:
Sounds good!